cuda_core/cuda/core/experimental/_memory.pyx

```diff
@@ -560,9 +562,63 @@ class DeviceMemoryResource(MemoryResource):
         When using an existing (current or default) memory pool, the returned
         device memory resource does not own the pool (`is_handle_owned` is
         `False`), and closing the resource has no effect.
+
+    IPC-Enabled Memory Resources
+    ----------------------------
```

Comment:
The indentation is off so the doc is not rendering correctly:
https://nvidia.github.io/cuda-python/pr-preview/pr-1020/cuda-core/latest/generated/cuda.core.experimental.DeviceMemoryResource.html#cuda.core.experimental.DeviceMemoryResource

Reply:
I had forgotten that there are finite number of reserved keywords that can be used as section titles in docstrings.
https://www.sphinx-doc.org/en/master/usage/extensions/napoleon.html#docstring-sections
I made a small tweak in commit 534b16a that should do the job. Well, we will hopefully re-work on the docs later...